dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonUtils_JSON.java

```diff
@@ -331,32 +321,27 @@ private static String jsonifyTraceAndMessage(int maxFieldLength, String wlpUserD
                 for (KeyValuePair k : extensions) {
                     String extKey = k.getKey();
                     if (extKey.endsWith(CollectorJsonHelpers.INT_SUFFIX)) {
-                        isFirstField &= !CollectorJsonHelpers.addToJSON(sb, LogTraceData.getExtensionNameKeyJSON(isMessageEvent, extKey), Integer.toString(k.getIntValue()),
-                                                                        false, true, false, isFirstField, true);
+                        jsonBuilder.addField(LogTraceData.getExtensionNameKeyJSON(isMessageEvent, extKey), k.getIntValue(), false);
```

Comment:
these need to be converted to strings (so they get surrounded by quotes) for backward compatibility